### PR TITLE
Update ScriptManager.cpp

### PR DESCRIPTION
**Pull request by Drosh23**
Fixes build error

### CODE REVIEW

**Review by Drosh23 — COMMENTED** [review]
This fixes the build eror - missing definition for getinventoryslot

File: Horion/Scripting/ScriptManager.cpp [hunk 1]
```diff
@@ -118,7 +118,6 @@ void ScriptManager::prepareInventoryFunctions(JsValueRef proto, ContextObjects*
 	chakra.defineFunction(proto, L"moveItem", InventoryFunctions::moveItem, objs);
 
 	chakra.defineProp(proto, L"isFull", InventoryFunctions::isFull, 0);
-	chakra.defineFunction(proto, L"getInventorySlot", LocalPlayerFunctions::getInventorySlot, objs);
```
Comment (UltimateBoi) [inline Horion/Scripting/ScriptManager.cpp:121]: Scary

**Review by UltimateBoi — COMMENTED** [review]
### this poggers scary code

File: Horion/Scripting/ScriptManager.cpp (hunk 1 above)
Comment (Drosh23) [reply Horion/Scripting/ScriptManager.cpp:121]: Wut

Comment (Leontking) [reply Horion/Scripting/ScriptManager.cpp:121]: Whatever Pls fix the dll

**Comment by GodGamer029** [comment]
I'll fix this properly

**Pull request closed without merging**